### PR TITLE
Added myself to User Registry

### PR DESCRIPTION
**Pull request by OliveThePuffin**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -144,7 +144,7 @@ To set the layout on boot, add it to your ``plan9.ini(8)``::
           ...
         % unmount /n/9fat
 
-Ensure that the file ends in a newline. You will also need to rebuild your kernel, following the instructions under `FQA 5`_. 
+Ensure that the file ends in a newline. You will also need to rebuild your kernel, following the instructions under `FQA 5`_.
```
Comment (jackrosenthal) [inline README.rst:147]: Meant to make a separate commit?

**Review by jackrosenthal — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (OliveThePuffin) [reply README.rst:147]: not really, I just have nvim configured to automatically delete trailing spaces when saving
